shacl/BrickEntityShapeBase.ttl

```diff
@@ -111,6 +119,40 @@ bsh:PointShape a sh:NodeShape;
     ) ;
     .
 
+bsh:SystemShape a sh:NodeShape;
+    sh:targetClass brick:System;
```

Comment:
I'm so sorry I missed this, but this should also be changed to `Collection` instead of `System`:  I think its just the `CollectionShape` and `brick:System`

Reply:
Then the rest of the use of "System" should be changed to "Collection"

Reply:
I'm a bit confused. If I change SystemShape to CollectionShape then there will be no SystemShape. Should I add an CollectionShape besides the SystemShape? Currently, there is no CollectionShape.

Reply:
We will want a CollectionShape in this file, yes. Brick actually generates a NodeShape for the `brick:System` class that is done automatically (see the last line):

```turtle
brick:System a owl:Class,
        sh:NodeShape ;
    rdfs:label "System" ;
    rdfs:subClassOf   brick:Collection ;
    skos:definition "A System is a combination of equipment and auxiliary devices (e.g., controls, accessories, interconnecting means, and termi­nal elements) by which energy is transformed so it performs a specific function such as HVAC, service water heating, or lighting. (ASHRAE Dictionary)."@en ;
    sh:property [ sh:or ( [ sh:class brick:Equipment ] [ sh:class brick:Point ] [ sh:class brick:Loop ] [ sh:class brick:System ] [ sh:class brick:Location ] [ sh:class brick:PV_Array ] ) ;
            sh:path brick:hasPart ] . 
```

We can definitely put a `SystemShape` in this file --- maybe even move the System shape definition to this file --- but we will need a `CollectionShape` nonetheless